pkg/commands/logging/azureblob/describe.go

```diff
@@ -103,27 +104,29 @@ func (c *DescribeCommand) Exec(_ io.Reader, out io.Writer) error {
 		}
 		return nil
 	}
-
+	lines := text.Lines{
```

Comment:
Sorry to be a pain, but can we ensure all the fields are also listed in the code in order (again, it makes it easier to find and add new fields in future). If you could do that across each file please.

Reply:
Done in https://github.com/fastly/cli/pull/698/commits/d79b71ec0b32c3121d76331fef143a8231e21671